### ISSUE TITLE
Create main types

### ISSUE THREAD

**Issue by ataias** [ataias/financial#2]
My plan for this project so far is for it to have a flow like:

- parse txt/csv data from selected brokers
- convert such data to the "Investment" type, to be implemented, likely:
```fsharp
type Investment =
  | Bond of Bond
  | Stock of Stock
  | Cash of Cash
```
- data of type `Investment` has the main info for the subtype, but coming from the broker it may only have info for the day it was download from. The idea is to have a type "Account" that has an array with previously read data and that will then append data to it
```fsharp
type Account = 
  { Description: string
    Bank: TBA
    Ledger: Array<TBA>
    Owner: string
    Interpolation: Interpolation }
```

In summary, this issue should create the type `Investment` and `Account` and update the existing code with them.
